test/DebtToken.t.sol

```diff
@@ -88,8 +88,38 @@ contract DebtTest is DebtTokenTest {
     function setUp() public override {
         super.setUp();
     }
+    
+    function testSuccess_totalAssets(uint128 assets, address receiver, uint24 deltaBlocks) public {
+        // Given: all neccesary contracts are deployed on the setup and last sync block is set
+        vm.assume(assets > 0);
+        vm.assume(deltaBlocks <= 13140000); //5 year
+        
+        pool.syncInterests();
+        vm.prank(address(pool));
+        // When: pool deposits assets
+        debt.deposit(assets, receiver);
+        uint64 interestRate = pool.interestRate();
+
+        vm.roll(block.number + deltaBlocks);
+
+        uint256 unrealisedDebt = calcUnrealisedDebtChecked(interestRate, deltaBlocks, assets);
+        emit log_uint(unrealisedDebt);
+        uint256 actualValue = assets + unrealisedDebt;
```

Comment:
actual & expected value should be switched. The expected is what you calculate, the actual is what you fetch from the contract.